### PR TITLE
enh: reduce front pg connection pool size

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

We increased the number of pods, but we haven't recuced the # of connections per instance. We're getting dangerously close to max out the DB, and we'e had some related errors.

### CODE REVIEW

**Review by PopDaph — APPROVED** [review]
🙏🏻 
(Could you link to an example of the errors we had?)

File: front/lib/resources/storage/index.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const frontSequelize = new Sequelize(
   {
     pool: {
       // Default is 5.
-      max: isDevelopment() ? 5 : 50,
+      max: isDevelopment() ? 5 : 40,
```
Comment (flvndvd) [inline front/lib/resources/storage/index.ts:14]: We could probably reduce even more.

**Review by flvndvd — APPROVED** [review]

**Comment by fontanierh** [comment]
@PopDaph  here's an example: https://app.datadoghq.eu/logs?query=unhandled%20&cols=host%2Cservice&event=AgAAAY7v3Iup_xmY3AAAAAAAAAAYAAAAAEFZN3YzSko2QUFEeWc0TjZCU0NEZ3dBTwAAACQAAAAAMDE4ZWYwMDEtNDk3MS00YWIxLWFmYzQtMjc4ZjU3ZjA3ZDJm&fromUser=true&index=%2A&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1713275112862&to_ts=1713361512862&live=true

File: front/lib/resources/storage/index.ts (hunk 1 above)
Comment (fontanierh) [reply front/lib/resources/storage/index.ts:14]: It's true that we removed a lot of the promise.all's. Likely safer to do it progressively though

**Pull request merged**